cupy/core/elementwise.pxi

```diff
@@ -514,7 +514,9 @@ cdef class ElementwiseKernel:
             raise ValueError("Can not use 'i' as a parameter name")
 
     def __call__(self, *args, **kwargs):
-        """Compiles and invokes the elementwise kernel.
+        """__call__(*args, **kwargs)
```

Comment:
What's the purpose of this change?

Reply:
Removed this change.
I intended to align with `ReductionKernel` but it was meaningless.  Ref. #808